editions/tw5.com/tiddlers/concepts/WikiText.tid

```diff
@@ -4,12 +4,16 @@ tags: Concepts Reference
 title: WikiText
 type: text/vnd.tiddlywiki
 
-~WikiText is a concise, expressive way of typing a wide range of text formatting, hypertext and interactive features. It allows you to focus on writing without a complex user interface getting in the way. It is designed to be familiar for users of [[MarkDown|http://daringfireball.net/projects/markdown/]], but with more of a focus on linking and the interactive features.
+~WikiText is a concise, expressive way of typing a wide range of text formatting, hypertext and interactive features. It allows you to focus on writing without a complex user interface getting in the way. It is designed to be familiar for users of [[Markdown]] but with more of a focus on linking and the interactive features.
 
-~WikiText can also be inserted to the text field using the [[Editor toolbar]].
+!! Elements
```

Comment:
I'm not sure that this heading is helpful; the word "elements" in line 15 is poorly chosen given the ambiguity with HTML elements. Since this is a short tiddler I don't think we need a heading at all.